sklearn/metrics/cluster/tests/test_unsupervised.py

```diff
@@ -185,25 +187,39 @@ def assert_raises_on_all_points_same_cluster(func):
                          rng.rand(10, 2), np.arange(10))
 
 
-def test_calinski_harabaz_score():
-    assert_raises_on_only_one_label(calinski_harabaz_score)
+def test_calinski_harabasz_score():
+    assert_raises_on_only_one_label(calinski_harabasz_score)
 
-    assert_raises_on_all_points_same_cluster(calinski_harabaz_score)
+    assert_raises_on_all_points_same_cluster(calinski_harabasz_score)
 
     # Assert the value is 1. when all samples are equals
-    assert_equal(1., calinski_harabaz_score(np.ones((10, 2)),
-                                            [0] * 5 + [1] * 5))
+    assert_equal(1., calinski_harabasz_score(np.ones((10, 2)),
+                                             [0] * 5 + [1] * 5))
 
     # Assert the value is 0. when all the mean cluster are equal
-    assert_equal(0., calinski_harabaz_score([[-1, -1], [1, 1]] * 10,
-                                            [0] * 10 + [1] * 10))
+    assert_equal(0., calinski_harabasz_score([[-1, -1], [1, 1]] * 10,
+                                             [0] * 10 + [1] * 10))
 
     # General case (with non numpy arrays)
     X = ([[0, 0], [1, 1]] * 5 + [[3, 3], [4, 4]] * 5 +
          [[0, 4], [1, 3]] * 5 + [[3, 1], [4, 0]] * 5)
     labels = [0] * 10 + [1] * 10 + [2] * 10 + [3] * 10
-    pytest.approx(calinski_harabaz_score(X, labels),
-                        45 * (40 - 4) / (5 * (4 - 1)))
+
```

Comment:
remove this blank line

Reply:
I added the blank line as Flake8 requires 2 blank lines before the next function definition

Reply:
Next line is not a function?

Reply:
I mistook it for the blank line after the pytest.approx() call

Reply:
I did not follow this comment - "I assume you've checked all the remaining calinski_harabaz_index are reasonable"

Reply:
Search the wrong name in the repo and make sure that you've corrected all of them (except for some special cases, e.g., we need to keep the original function for 2 versions)

Reply:
Use `git grep harabaz` for instance.